apps/typescript/enums-vs-union-types/src/app/app.component.ts

```diff
@@ -1,14 +1,14 @@
 import { Component, computed, signal } from '@angular/core';
 
-enum Difficulty {
-  EASY = 'easy',
-  NORMAL = 'normal',
-}
+type Difficulty = 'Easy' | 'Normal';
+type Direction = 'left' | 'right';
 
-enum Direction {
-  LEFT = 'left',
-  RIGHT = 'right',
-}
+type DirectionLabelMap = { [key in Direction]: string };
+
+const directionLabels: DirectionLabelMap = {
```

Comment:
you can also use a record here

Reply:
yes indeed! But then I will not use a mapped type which is one of the goal of the challenge. 
I added the record solution but commented it out for those who are wondering.
Thank you!